backend/amd64/emit.mlp

```diff
@@ -65,10 +65,13 @@ let cfi_adjust_cfa_offset n =
 let emit_debug_info ?discriminator dbg =
   emit_debug_info_gen ?discriminator dbg D.file D.loc
 
-let emit_debug_info_linear i =
-  match i.fdo with
-  | None -> emit_debug_info i.dbg
+let emit_debug_info_instr ~pad fdo dbg =
```

Comment:
I don't quite like `~pad` as it's pretty non-descriptive. Would it instead be possible to have a second function,
`emit_debug_info_synthetic_block` what will always write the negative discriminator and keep `emit_debug_info_linear` as-is?

Reply:
agree that `pad` is a bad name, I'll try to find a better one and refactor this little function. I don't want to duplicate the logic that decides which location to emit.